blog/2021-09-17-announcing-stryker-net-1.md

```diff
@@ -0,0 +1,265 @@
+
+## 🚀 Whats new?
+
+An exciting set of new features has been added with this release! Let's walk trough them all:
+
+### Allow failing tests
+
+It's now allowed to start a mutation test run even with failing tests. Stryker will try to make the best of the situation by marking mutants that are covered by `initially failing tests` as `survived`.
+
+### Mutant filtering
+
+It's now possible to filter mutants at source code level. This gives the most fine grained level of control.
+
+The syntax for the comments is: `Stryker [disable|restore][once][all| mutator list][: reason for disabling]`
+
+`// Stryker disable all` Disables all mutations from that line on.
+
+`// Stryker restore all` re-enables all mutations from that line on.
+
+`// Stryker disable once all` will only disable mutations on the next line.
+
+`// Stryker disable once Arithmetic,Update` will only disable Arithmetic and Update mutations on the next line
+
+Example:
+
+```csharp
+var i = 0;
+var y = 10;
+// Stryker disable all : reason for disable
+i++; // won't be mutated
+y++; // won't be mutated
+// Stryker restore all
+i--; // will be mutated
+// Stryker disable once all
+y--; // won't be mutated
+i++; // will be mutated
+// Stryker disable once Arithmetic
+y++; // will be mutated
+// Stryker disable once Arithmetic,Update
+i--; // won't be mutated
+```
+
+### Ignore mutations
+
+The ignore mutations option has been extended to offer more fine grained control. Before v1.x it was possible to ignore complete mutators. Now it's possible to ignore specific mutations inside these mutators as well.
+
+Example:
+
+```json
+"stryker-config": {
+    "ignore-mutations": [
+        "linq.First",
+        "linq.Sum"
+    ]
+}
+```
+
+> Note: this only works for Linq mutations for now.
+
+All possible mutations:
+
+```
+None,
+Distinct,
+Reverse,
+Average,
+AsEnumerable,
+OrderBy,
+OrderByDescending,
+FirstOrDefault,
+First,
+SingleOrDefault,
+Single,
+Last,
+All,
+Any,
+Skip,
+Take,
+SkipWhile,
+TakeWhile,
+Min,
+Max,
+Sum,
+Count,
+ThenBy,
+ThenByDescending,
+Union,
+Intersect,
+Concat,
+Except
+```
+
+### MsBuild path option
+
+By default stryker tries to autodiscover msbuild on your system. If stryker fails to discover msbuild you may supply the path to msbuild manually with this option.
+
+Example:
+
+```shell
+--msbuild-path "c://MsBuild/MsBuild.exe"
+```
+
+### Target framework
+
+If the project targets multiple frameworks, it is now possible to specify the particular framework to build against. If you specify a non-existent target, Stryker will build the project against a random one (or the only one if so).
+
+Example:
+
+```json
+{
+  "stryker-config": {
+    "target-framework": "netcoreapp3.1"
+  }
+}
+```
+
+### Filter test cases
+
+A long awaited feature has finally found its way into Stryker! It is now possible to exclude some test cases. For example if you have long running integration tests in your unit test project they can be disabled for Stryker. This will improve runtime.
```

Comment:
```suggestion
A long awaited feature has finally found its way into Stryker! It is now possible to exclude some test cases. For example if you have long running integration tests in your unit test project they can be disabled for Stryker. You can use this to for example disable your integration tests if they are in the same project as your unit tests.
```

Reply:
It says the same in the sentence before this